mqtt/mqtt-bridge/src/persist/waking_state/waking_store.rs

```diff
@@ -0,0 +1,97 @@
+use std::collections::hash_map::Entry;
+use std::{boxed::Box, collections::HashMap, task::Waker};
+
+use bincode::{deserialize, serialize, ErrorKind};
+use mqtt3::proto::Publication;
```

Comment:
https://github.com/birktj/rocksbin-db this crate seems small and neat. what do you think?

Reply:
You might have more developed opinions here, but my intuition is that using unpopular crates is bad from a security perspective.

Reply:
this is correct. but in this case, the library is so small and looks like does exactly what we need. it is tempting to take it :)
Alternatively, I believe we should make similar abstraction to support serde for rocksdb

Reply:
I am thinking it is better to write a simple wrapper ourselves. If we could trust the package to not have bugs or update rocksdb (which looks to be using an old version) then I would use it. Also, the ```unwarp()``` spelling in the README makes me quite suspect